Extension/src/LanguageServer/client.ts

```diff
@@ -2976,6 +3027,41 @@ export class DefaultClient implements Client {
         }
     }
 
+    public async handleGenerateDoxygenComment(line: number | undefined, column: number | undefined): Promise<void> {
+        const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+        if (!editor) {
+            return;
+        }
+        const isCodeAction: boolean =  (line !== undefined && column !== undefined);
+        const params: GenerateDoxygenCommentParams = {
+            uri: editor.document.uri.toString(),
+            position: isCodeAction ? new vscode.Position(line ?? 0, column ?? 0) : editor.selection.active,
+            isCodeAction: isCodeAction
+        };
+        await this.awaitUntilLanguageClientReady();
+        const currentFileVersion: number | undefined = openFileVersions.get(params.uri);
+        if (currentFileVersion === undefined) {
+            return;
+        }
+        const result: GenerateDoxygenCommentResult | undefined = await this.languageClient.sendRequest(GenerateDoxygenCommentRequest, params);
+        if (result?.fileVersion !== undefined &&
```

Comment:
if you add `result !== undefined` as the first check you should be able to change all the `?.` to a `.`.